proof/refine/ARM/Tcb_R.thy

```diff
@@ -2981,8 +2986,7 @@ lemma get_sc_released_corres:
    apply (wpsimp wp: no_ofail_gets_the readRefillReady_no_ofail)
   apply (clarsimp simp: sc_released_def readRefillReady_def readSchedContext_def
                  dest!: readObject_misc_ko_at')
-  apply normalise_obj_at'
-  apply (subgoal_tac "sc_active sc = (0 < scRefillMax v')")
+  apply (subgoal_tac "sc_active sc = (0 < scRefillMax sca)")
```

Comment:
Maybe not worth it, but if may be nice to `rename_tac` this `sca` too, especially considering it's changed its generated name